### ISSUE TITLE
[Adaptive Cards]: "Microsoft" Link is not adopting the Aquatic theme of high contrast.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7866]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Settings >Accessibility> Contrast themes> Aquatic themes.

Repro Steps:
1. Open above URL with V- credential.
2. Navigate to 'Microsoft' link using tab key in the "Navigation Landmark".

3. Observe the issue here.

Actual Result:
"Microsoft" Link is not clearly visible to the user in the aquatic/Dessert mode settings.

Expected Result:
Microsoft Link should adopt the Aquatic/Dessert theme settings so that it will be clearly visible to the user.

Observation:
Similar issue on "Get started" link and "Adaptive Card" link.
4.Adaptive Card text is not adopting the Aquatic settings
4.get started link in aquatic
Same issue exists throughout the application wherever the similar controls present in the adaptive cards.

User Experience:
Visual disable user and the normal user will get impacted as its very difficult to identify the link on the page as its not that clearly visible and not adopting the theme settings.

MAS Reference link:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="4 microsoft link in Aquatic settings" src="https://user-images.githubusercontent.com/104482612/190348641-84c03ae6-73f1-4bc3-b708-34b835ae4f71.png">

<img width="870" alt="4 get started link in aquatic" src="https://user-images.githubusercontent.com/104482612/190348675-6ae0b61e-9594-4b6b-b11c-cb0f6cc79155.png">

<img width="961" alt="4 Adaptive Card text is not adopting the Aquatic settings" src="https://user-images.githubusercontent.com/104482612/190358022-fd3db419-2a4f-4c0c-9219-3b485e07ca07.png">

<img width="960" alt="4 Microsoft in dessert theme" src="https://user-images.githubusercontent.com/104482612/190358055-8cd9e2fa-1341-4b4c-bfd6-6dd084b89e32.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by anna-dingler** [comment]
Hi @Susmitha474,

I have opened a PR to address the high contrast issues pointed out here.

I did want to note a couple of changes that I did not address.
 - Since the Microsoft link is a logo, it is not subject to the same contrast requirements of text-only links (https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.04.03). Therefore, I did not make any changes to the logo.
 - The "Get Started" link that you requested text color changes for is actually a button. To make this clear, I added a border to the button instead, and it now matches the contrast settings.
 
<img width="394" alt="image" src="https://user-images.githubusercontent.com/98650930/211086470-c32c28ca-6bd2-4daa-8841-ad8fa449602b.png">

Thanks!
Anna
